tera-templates/header/topbar.html

```diff
@@ -0,0 +1,97 @@
+{#
+    This is the unchanging top bar that is on every single page.
+    The only piece of context it can take is `search_query`, which should
+    be a string and will populate the search field if it exists
+#}
+<div class="nav-container">
+    <div class="container">
+        <div class="pure-menu pure-menu-horizontal" role="navigation" aria-label="Main navigation">
+            <form action="/releases/search" method="GET" class="landing-search-form-nav">
+                {# The search bar #}
+                <div id="search-input-nav">
+                    <label for="nav-search">
+                        <i class="fa fa-fw fa-search"></i>
+                    </label>
+
+                    {# If there is a search query, put it in the search bar #}
+                    <input id="nav-search" name="query" type="text" aria-label="Find crate by search query"
+                        placeholder="Find crate" {%- if search_query %} value="{{ search_query }}" {%- endif %}>
+                </div>
+
+                {# The top-left logo and name #}
+                <a href="/" class="pure-menu-heading pure-menu-link" aria-label="Docs.rs">
+                    <i class="fa fa-cubes fa-fw"></i> Docs.rs
+                </a>
+
+                <ul class="pure-menu-list">
+                    {# The releases button #}
```

Comment:
This comment seems pretty redundant with the title of the link.
```suggestion
```

Reply:
Agreed, I'm actually trying to figure out a good way to automate this section so it's not just a copy+paste